halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/providers/appengine/AppengineCommandProperties.java

```diff
@@ -37,4 +37,8 @@
             + " against a known_hosts file.";
     static final String GCLOUD_RELEASE_TRACK = "The gcloud release track (ALPHA, BETA, or STABLE) that Spinnaker"
             + " will use when deploying to App Engine.";
+    static final String SERVICES = "The list of services to which Spinnaker should limit its indexing.";
```

Comment:
I realize we don't do this yet, but is it possible to use these to prevent a user from modifying/deploying a certain service or version in an account? We do the same for namespaces & kinds in k8s for both the indexing & deploying.

Reply:
Interesting idea. Quickly looking at DeployAppengineAtomicOperation this appears to be doable. I've created an issue to track this: https://github.com/spinnaker/spinnaker/issues/3590